packages/contract/package.json

```diff
@@ -11,7 +11,7 @@
   "version": "4.1.5",
   "main": "index.js",
   "scripts": {
-    "compile": "mkdir -p dist && browserify ./index.js -o ./dist/truffle-contract.js && uglifyjs ./dist/truffle-contract.js -o ./dist/truffle-contract.min.js",
+    "compile": "sh -c \"mkdir -p ./dist\" && browserify ./index.js -o ./dist/truffle-contract.js && uglifyjs ./dist/truffle-contract.js -o ./dist/truffle-contract.min.js",
```

Comment:
Can you pull out this commit and open a separate PR targeting `develop` in #2780?

Reply:
Nm, got it #2784.